docs/framework/install/on-windows-10.md

```diff
@@ -14,21 +13,21 @@ ms.assetid: 85cda1d5-c6g4-4eb5-93e6-4f478de07ff7
 
 # Install the .NET Framework on Windows 10
 
-You may need the .NET Framework to run an application on Windows 10. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+You may need the .NET Framework to run an app on Windows 10. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
 
-## .NET Framework 4.6.2
+## .NET Framework 4.7
 
-The .NET Framework 4.6.2 is included with Windows 10 Anniversary Update. 
+The .NET Framework 4.7 is included with the [Windows 10 Creators Update](https://www.microsoft.com/software-download/windows10).
 
-If you are on an earlier version of Windows 10, you will need to upgrade to [Windows 10 Anniversary Update](https://www.microsoft.com/software-download/windows10) to get the .NET Framework 4.6.2.
+If you're on an earlier version of Windows 10, you'll need to upgrade to [Windows 10 Creators Update](https://www.microsoft.com/software-download/windows10) to get the .NET Framework 4.7.
```

Comment:
[I thought you can install .Net Framework 4.7 on Win 10 Anniversary.](https://blogs.msdn.microsoft.com/dotnet/2017/05/02/announcing-the-net-framework-4-7-general-availability/)